src/oph/ehoks/db/db_operations/oppija.clj

```diff
@@ -21,38 +16,3 @@
 
 (defn insert-oppija! [oppija]
   (db-ops/insert-one! :oppijat (db-ops/to-sql oppija)))
-
```

Comment:
Tämä olikin hyvä siirto

Reply:
Jep, olivat alunperinkin väärässä paikassa. Pitää vielä katsella etten unohtanut jonnekin muuallekin näitä.